boards/arm/nrf5340_audio_dk_nrf5340/Kconfig

```diff
@@ -0,0 +1,55 @@
+# nRF5340 Audio DK board configuration
+
+# Copyright (c) 2019 Nordic Semiconductor ASA
```

Comment:
This is a project that has been worked on since 2018, the year reflects when the file was created. Do you mean that all files in one commit should have the same year?

Reply:
I was just pointing out.